docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -7,11 +7,54 @@ sidebar_label: Enhanced Project Workflow
 import Tabs from "@theme/Tabs"
 import TabItem from "@theme/TabItem"
 
-:::caution
-This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::info
+This is a **BETA** feature and, as such, is currently under active development.
+
+It can be safely used but there may be still be some missing features, check the [roadmap section](#roadmap-and-future-improvements) below to find out more.
```

Comment:
```suggestion
It can be safely used but there may still be some missing features: check the [roadmap section](#roadmap-and-future-improvements) below to find out more.
```